class.jetpack.php

```diff
@@ -5383,21 +5388,21 @@ function verify_xml_rpc_signature() {
 			$body = null;
 		}
 
-		$signature = $jetpack_signature->sign_current_request(
-			array( 'body' => is_null( $body ) ? $this->HTTP_RAW_POST_DATA : $body, )
-		);
+		$signature = $jetpack_signature->sign_current_request( array(
+			'body'      => is_null( $body ) ? $this->HTTP_RAW_POST_DATA : $body,
+			'nonce'     => $nonce,
+			'timestamp' => $timestamp,
+			'token'     => $token,
```

Comment:
@kwight @jeherve `sign_current_request()` expects this data to be available in `$_GET` as well, so I had to extend the overrides to make it work for function-parameter-only calls.

Reply:
Goodness, the stain spreads. Fighting the good fight of backwards compatibility 👍

Reply:
I should have phrased that better.

Unless explicitly passed on, `sign_current_request()` expects this data to be available in $_GET, so I had to extend the overrides to make it work for function-parameter-only calls.

Reply:
Noting that we're no longer messing with `verify_xml_rpc_signature`.